src/main/java/io/fabric8/runsh/RunShLoader.java

```diff
@@ -1,25 +1,90 @@
 package io.fabric8.runsh;
 
-import java.io.*;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.CopyOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.attribute.PosixFilePermission;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
 
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.FilenameUtils;
-import org.apache.commons.io.IOUtils;
 
 /**
  * Load and export run-java.sh script
  */
 public class RunShLoader
 {
-    public static final String[] LOCATION_RUN_SCRIPTS = new String[] {
-            "/run-java-sh/fp-files/container-limits",
-            "/run-java-sh/fp-files/debug-options",
-            "/run-java-sh/fp-files/java-default-options",
-            "/run-java-sh/fp-files/run-java.sh"
-    };
+    private static final String RUN_SCRIPT = "run-java.sh";
+    private static final String LOCATION_RUN_SCRIPT = "/run-java-sh/fp-files/" + RUN_SCRIPT;
+    private static final String LOCATION_README = "/run-java-sh/readme.md";
```

Comment:
Maybe I'll get consistent with the case of the README

Reply:
agreed, would have been better to call it README.md, but its no stored and referenced from the images are `readme.md`, so for backward compatibilities sake, I'll keep it. In the Java code its only read under this name, not stored.